rest-client/src/main/java/io/apicurio/registry/client/RegistryRestClientImpl.java

```diff
@@ -438,4 +447,13 @@ public void setNextRequestHeaders(Map<String, String> headers) {
     public Map<String, String> getHeaders() {
         return requestHeaders.get();
     }
+
+    private String encodeURIComponent(String value) {
+        try {
+            //TODO what to use here ASCII or UTF_8 ??
```

Comment:
Since UTF-8 legacily supports ASCII, for me, +1 to UTF-8. That said if we end up using only US-ASCII characters (no accents etc) and I think that's the case here, then it shouldn't matter.

Reply:
+1 to UTF-8 for me as well.  That's generally considered the right default to use in most situations.